Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt

```diff
@@ -62,8 +62,13 @@ object Libs {
             const val uiTest = "androidx.compose.ui:ui-test-junit4:$version"
         }
 
+        object Activity {
+            const val activityCompose = "androidx.activity:activity-compose:1.3.0-SNAPSHOT"
+        }
+
         object Lifecycle {
             private const val version = "2.3.0-beta01"
+            const val viewModelCompose = "androidx.lifecycle:lifecycle-viewmodel-compose:1.0.0-SNAPSHOT"
```

Comment:
Don't you want to use the compose $version variable here?

Reply:
This doesn't follow Compose version. This will be `1.0.0-alpha01` in the next release